src/screens/Home/hooks/usePeople.ts

```diff
@@ -6,27 +6,36 @@ import { People } from '../types';
 
 interface UsePeople {
   people: People[];
+  isLoading: boolean;
+  isError: boolean;
 }
 
 const usePeople = (): UsePeople => {
   const [people, setPeople] = useState<People[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isError, setIsError] = useState<Error | undefined>(undefined);
```

Aja marico, hay una vaina (un objeto) en Javascript que se llama [Error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error). Es un objeto "especial" porque puedes `throw/catch` 

```
const aFunctionThatThrowsAnError = () => {
   throw new Error()
}

try {
   aFunctionThatThrowsAnError
} catch (error) { // Este es el error que le haces "throw"
}
```

en el caso de `fetchPeople`, nosotros lo cambiamos precisamente a `throw new Error()` por lo tanto el `catch()` recibe el error: `catch(error)`

ese `Error` object lo usamos mucho para describir un error, tu puedes hacerlo asi: `new Error('An error occurred when attempting to fetch People')`

Y eso lo puedes loggear: `console.error(error)` o lo puedes usar para mostrarselo al usuario: `<Text>{error.message}</Text>

Reply:
Listo, no tengo que crear un flag sino que uso ese object.